lib/src/scripts/configure.js

```diff
@@ -3,12 +3,16 @@ const path = require('path');
 const shell = require('shelljs');
 const parse = require('yargs-parser');
 
-const { appDirectory, iterateUserEnvironment } = require('../util');
+const { appDirectory, log, iterateUserEnvironment } = require('../util');
 
 const args = parse(process.argv.slice(2));
 const buildDir = args.buildPath || 'dist';
 
+log.header('Configuring project...');
+
 iterateUserEnvironment((key, value) => {
+  log.debug(`Replacing key: ${key}`);
```

Comment:
@byhag can we easily determine if we try to replace a key which does not exist in `env.js`? Or the opposite even? Both of those would be useful but I don't know off the top of my head if that is possible or not.

Reply:
@mauleb To the first question, yes. To the second, I don't think so without reworking how `env.js` works.